msvc/scripts/csproj.tmpl

```diff
@@ -13,62 +12,60 @@
     <HostPlatform Condition=" '$(HostPlatform)' == '' and '$(OS)' == 'Windows_NT'">win32</HostPlatform>
     <HostPlatform Condition=" '$(HostPlatform)' == '' and '$(OS)' == 'Unix' and $([System.IO.File]::Exists('/usr/lib/libc.dylib'))">darwin</HostPlatform>
     <HostPlatform Condition=" '$(HostPlatform)' == '' and '$(OS)' == 'Unix'">linux</HostPlatform>
-    <OutputPath>@OUTPUTDIR@</OutputPath>
-    <IntermediateOutputPath>obj-@OUTPUTSUFFIX@</IntermediateOutputPath>
     <GenerateTargetFrameworkAttribute>false</GenerateTargetFrameworkAttribute>
     @NOSTDLIB@
     @METADATAVERSION@
     @STARTUPOBJECT@
     @NOCONFIG@
     @ALLOWUNSAFE@
     <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>
-    </RootNamespace>
+    <RootNamespace></RootNamespace>
     <AssemblyName>@ASSEMBLYNAME@</AssemblyName>
-    <TargetFrameworkVersion>v@FX_VERSION</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v@FX_VERSION@</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
-@SIGNATURE@
+    @SIGNATURE@

```

Comment:
this shouldn't be indented, the variable that gets inserted here already has spaces in it...